File: Problem 9/sol.js
function pangrams(s) {
    const alphabet = "abcdefghijklmnopqrstuvwxyz ";
    s = s.toLowerCase();
    
    for(let ind in alphabet){
        if(s.indexOf(alphabet[ind])=="-1")
            return "not pangram";
    }
    
    return "pangram";

}

File: Problem 9/sol.cpp
string pangrams(string s) {
    string alphabet = "abcdefghijklmnopqrstuvwxyz ";
    
    for(char c : alphabet){
        size_t encontrado = s.find(c);
        if(encontrado==std::string::npos ){
            size_t encontradoMayus = s.find(std::toupper(c));
            if(encontradoMayus==std::string::npos ){
              return "not pangram";
            }
        }
    }
    return "pangram";
}